packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -4,90 +4,111 @@
 
 import React, { useCallback } from 'react';
 import { useBooleanState } from 'fxa-react/lib/hooks';
+import { useQuery } from '@apollo/client';
+import AppErrorDialog from 'fxa-react/components/AppErrorDialog';
+import LoadingSpinner from 'fxa-react/components/LoadingSpinner';
 import UnitRow from '../UnitRow';
 import UnitRowWithImage from '../UnitRowWithImage';
 import Modal from '../Modal';
-
-const MOCK_ACCOUNT_DATA = {
-  displayName: 'Fred Flinstone',
-  avatarUrl: null,
-  passwordCreated: 123456789,
-  emails: [
-    {
-      email: 'user@example.com',
-      isPrimary: true,
-      verified: true,
-    },
-    {
-      email: 'user2@example.com',
-      isPrimary: false,
-      verified: true,
-    },
-  ],
-};
+import { GET_ACCOUNT, accountData } from './gql';
 
 export const Settings = () => {
-  const primaryEmail =
-    MOCK_ACCOUNT_DATA.emails.find((email) => email.isPrimary === true) ||
-    MOCK_ACCOUNT_DATA.emails[0];
+  const { loading, error, data } = useQuery(GET_ACCOUNT);
   const [modalRevealed, revealModal, hideModal] = useBooleanState();
   const onSecondaryEmailConfirm = useCallback(() => {
     console.log('confirmed - resend verification code');
     hideModal();
   }, [hideModal]);
 
+  if (loading) {
+    return (
+      <LoadingSpinner
+        className="bg-grey-20 flex items-center flex-col justify-center h-screen select-none"
+        svgClassName="w-10 h-10 animate-rotate animate-timing-linear animate-duration-800 animate-repeat-infinite"
+      />
+    );
+  }
+
+  if (error) {
+    return (
+      <AppErrorDialog
+        data-testid="error-dialog"
+        classes={{
+          base: 'bg-grey-20 flex items-center flex-col justify-center h-screen',
+          container: 'text-center max-w-lg',
+          header: 'text-grey-600 font-header text-lg font-bold mb-3',
+          text: 'text-grey-400',
```

Comment:
We really need to figure out what we're doing with Tailwind in `fxa-react` for storybook. These could just be default classes instead 😐

Reply:
I am happy making these default classes. I realize we talked about making them default unless they are shared. I forgot!